File: app/src/main/java/com/example/tourethiopia/travelagency/TravelData.java
package com.example.tourethiopia.travelagency;

import com.example.tourethiopia.R;

public class TravelData {

    private int name;
    private int hotelimagersourceid;
    private float rating;

    public TravelData(int name, int hotelimagersourceid, float rating) {
        this.name = name;
        this.hotelimagersourceid = hotelimagersourceid;
        this.rating = rating;
    }

    public static final com.example.tourethiopia.travelagency.TravelData[] TRAVEL_DATA= {new com.example.tourethiopia.travelagency.TravelData(R.string.imagin_travellname,
            R.drawable.imagintravel, (float)4.5),
            new com.example.tourethiopia.travelagency.TravelData(R.string.welcom_travellname, R.drawable.welcometour,(float) 4),
            new com.example.tourethiopia.travelagency.TravelData(R.string.aman_travellname, R.drawable.amerantravel,(float) 3.5),
            new com.example.tourethiopia.travelagency.TravelData(R.string.base_travellname, R.drawable.basetravel,(float) 4),
            new com.example.tourethiopia.travelagency.TravelData(R.string.yema_travellname, R.drawable.yamatravel,(float) 4),
            new com.example.tourethiopia.travelagency.TravelData(R.string.ethio_travellname, R.drawable.ethiotravel,(float) 5),
            new com.example.tourethiopia.travelagency.TravelData(R.string.fklm_travellname, R.drawable.fklm,(float) 4.5),
            new com.example.tourethiopia.travelagency.TravelData(R.string.abba_travellname, R.drawable.abbatravel,(float) 4),

    };

    public int getName() {
        return name;
    }

    public int getHotelimagersourceid() {
        return hotelimagersourceid;
    }

    public float getRating() {
        return rating;
    }
}

File: app/src/main/java/com/example/tourethiopia/travelagency/TravelData.kt
package com.example.tourethiopia.travelagency

import com.example.tourethiopia.R

class TravelData(val name: Int, val hotelimagersourceid: Int, val rating: Float) {

    companion object {
        val TRAVEL_DATA = arrayOf(TravelData(R.string.imagin_travellname,
                R.drawable.imagintravel, 4.5.toFloat()),
                TravelData(R.string.welcom_travellname, R.drawable.welcometour, 4.toFloat()),
                TravelData(R.string.aman_travellname, R.drawable.amerantravel, 3.5.toFloat()),
                TravelData(R.string.base_travellname, R.drawable.basetravel, 4.toFloat()),
                TravelData(R.string.yema_travellname, R.drawable.yamatravel, 4.toFloat()),
                TravelData(R.string.ethio_travellname, R.drawable.ethiotravel, 5.toFloat()),
                TravelData(R.string.fklm_travellname, R.drawable.fklm, 4.5.toFloat()),
                TravelData(R.string.abba_travellname, R.drawable.abbatravel, 4.toFloat()))
    }

}